docs/examples/parser_sym_manager.rst

```diff
@@ -0,0 +1,10 @@
+Parser
```

Comment:
This should maybe not have the same title as `parser.rst`? Because then both show up as `Parser` entry in the page rendered from `examples.rst`.